host-bin/enter-chroot

```diff
@@ -434,10 +435,16 @@ elif [ ! -f "$shares" ]; then
 
 # Share the downloads folder of the current user's profile
 downloads ~/Downloads
+# Share the "My Files" folder of the current user's profile
+myfiles ~/MyFiles
```

Comment:
I don't think we should be adding this by default, especially on older systems that don't have the directory.

Reply:
Makes sense -- added another commit removing the default, feel free to squash them if preferred!